package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-merge": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "npm run clean && npm run build && npm run test",
+    "prepare": "npm run clean && npm run build && npm run test",
```

Comment:
See the discussion about `prepublish` and `prepare` here:
https://docs.npmjs.com/misc/scripts#prepublish-and-prepare

We want our code to be built after an `npm install`.  Even though the `prepublish` step was used previously to accomplish that, we must now use `prepare`, as the behaviour of `prepublish` will change (it will only called before a `publish`, like the name suggests).